common/src/main/java/org/apache/drill/common/config/DrillConfig.java

```diff
@@ -44,26 +46,33 @@
 import com.google.common.collect.ImmutableList;
 import com.typesafe.config.Config;
 import com.typesafe.config.ConfigFactory;
+import com.typesafe.config.ConfigRenderOptions;
 
 public final class DrillConfig extends NestedConfig{
-//  private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(DrillConfig.class);
+  private static final Logger logger = getLogger(DrillConfig.class);
```

Comment:
To get rid of the unnecessarily verbose qualified names in that boilerplate code (which, when having any unnecessary verbosity, adds unnecessary risk of not noticing mistakes like this:

public class TestMultiInputAdd extends PopUnitTestBase {

//    private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(TestMathFunctions.class);

)